pipeline/dbt/models/intermediate/sources/mes_aides/permis_velo/_mes_aides__models.yml

```
@@ -0,0 +1,57 @@
+version: 2
+
+models:
+  - name: int_mes_aides__adresses_permis_velo
```

```suggestion
  - name: int_mes_aides_permis_velo__adresses
```

ou

```suggestion
  - name: int_mes_aides_aides__adresses
```

selon ce que tu choisis de garder